ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -724,7 +724,8 @@ def _gather_event_display_context_from_workflow_crawling(
             workflow_display = get_workflow_display(workflow_class=workflow_class)
             return workflow_display.get_event_display_context()
 
-        except Exception:
+        except (ModuleNotFoundError, ValueError) as e:
+            logger.exception("Failed to load workflow from module %s: %s", module_path, e)
```

Comment:
- Don't need to include `e` in the error message, will be included.
- Only include `ModuleNotFoundError`s, not `ValueError`s